### PR TITLE
fix elements layout for surrending dialog

### PR DESCRIPTION
**Pull request by undef21**
resolve issues #2259, #2262

### CODE REVIEW

File: src/fheroes2/battle/battle_dialogs.cpp [hunk 1]
```diff
@@ -608,8 +608,8 @@ bool Battle::DialogBattleSurrender( const HeroBase & hero, u32 cost, const Kingd
 
     const int icn = conf.ExtGameEvilInterface() ? ICN::SURRENDE : ICN::SURRENDR;
 
-    fheroes2::Button btnAccept( pos_rt.x + 90, pos_rt.y + 150, icn, 0, 1 );
-    fheroes2::Button btnDecline( pos_rt.x + 295, pos_rt.y + 150, icn, 2, 3 );
+    fheroes2::Button btnAccept( pos_rt.x + 91, pos_rt.y + 152, icn, 0, 1 );
+    fheroes2::Button btnDecline( pos_rt.x + 295, pos_rt.y + 152, icn, 2, 3 );
```
Comment (ihhub) [inline src/fheroes2/battle/battle_dialogs.cpp:612]: The distance from left edge of the window to left button is 75 pixels while for the right - 65.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @undef21 are buttons on the same position as in the original game?

File: src/fheroes2/battle/battle_dialogs.cpp (hunk 1 above)
Comment (undef21) [reply src/fheroes2/battle/battle_dialogs.cpp:612]: Yes.
I was saw first screenshot from issue #2259.

compare

Base OG screenshot from issue. The part over red line is from fheroes2.

**Comment by ihhub** [comment]
@undef21 a huge thanks for fixing these issues! Good work!

**Pull request merged**